server/msteams/client.go

```diff
@@ -145,6 +146,22 @@ type GraphAPIError struct {
 	Message string `json:"message"`
 }
 
+type ChatAttachmentUser struct {
+	UserIdentityType string `json:"userIdentityType"`
+	ID               string `json:"id"`
+	DisplayName      string `json:"displayName"`
+}
+
+type ChatAttachmentSender struct {
+	User ChatAttachmentUser `json:"user"`
+}
+
+type ChatAttachment struct {
```

Comment:
```suggestion
type ChatMessageAttachment struct {
```
I think it would be more suitable. Let me know your thoughts.